docs/markdown/snippets/cython-limited-api.md

```diff
@@ -0,0 +1,6 @@
+## `limited_api` support for Cython extension modules
+
+Using the `limited_api` keyword when declaring a Python `extension_module`
+now causes Meson to define the `CYTHON_LIMITED_API` symbol. Cython's support
+for Python's Limited API was added in Cython version 3. If Meson detects a
+version of Cython earlier than this, a warning is raised.
```

Comment:
I think we don't need to document in the release notes, under what circumstances we emit an error.

Reply:
Ok, I'll remove the release note snippet.

Reply:
Sorry. To be clear, it's a new feature and needs a release notes snippet.

The question is how much detail should the snippet include. Usually we describe the new feature. It's unusual to *also* describe what circumstances emit a warning. The primary documentation is probably a better place for that.